CMakeLists.txt

```diff
@@ -9,50 +9,68 @@
 cmake_minimum_required(VERSION 3.10 FATAL_ERROR)
 project(sgct)
 
-include(support/cmake/copy_files.cmake)
-include(support/cmake/copy_sgct_dynamic_libraries.cmake)
-include(support/cmake/set_compile_options.cmake)
-include(support/cmake/disable_external_warnings.cmake)
-
-set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} ${PROJECT_SOURCE_DIR}/support/cmake/modules)
+if(POLICY CMP0056)
+     cmake_policy(SET CMP0056 NEW)

```

Comment:
Coding style
We use 2 space increment for CMake files

Reply:
ahh right...